test/types/collections/indices.good

```diff
@@ -26,10 +26,5 @@ d
 0..1
 0..3
 0..7
-{
- (1, 1)
- (2, 2)
- (3, 3)
-}
-
+(1, 1) (2, 2) (3, 3)
```

Comment:
I'm trying to understand this change: I would expect that the way we print out the domain here shouldn't change because you're using the new flag?

Reply:
This was in the OP as well.  Because we're using the new flag, we're getting the iterator rather than the "return a copy of the original domain", so this is printing out that iterator's yielded values.

Reply:
Ah ok. My mental framework was stuck in the world where the flag was used for opting out of the new behavior until you port the code.